cloud/pkg/csidriver/controllerserver.go

```diff
@@ -108,7 +108,11 @@ func (cs *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 	}
 
 	klog.V(4).Infof("create volume result: %v", result)
-	data := result.GetContent().(string)
+	data, ok := result.GetContent().(string)
+	if !ok {
+		klog.Errorf("malformed content was received: %s", data)
+		return nil, errors.New(data)
```

Comment:
LGTM, but if the value type is not a string, data here will be an empty string? it's better to add more info about data.

Reply:
How about this? @AdamKorcz :)
```
data, ok := result.GetContent().(string)
if !ok {
	klog.Errorf("content is not string type: %v", result.GetContent())
	return nil, fmt.Errorf("content type %T is not string", result.GetContent())
}
```

Reply:
Agree :) But seems that the returned error and klog printed error are duplicated, we can fix it in follow up PRs.